en/docs/learn/deploying-the-sample-app.md

```diff
@@ -453,15 +455,145 @@ The next step is to configure the service provider.
+on your browser.
+
+For example,
+`                     http://localhost:8080/pickup-dispatch.com/home.jsp                 .         `
+    
+### Configuring Service Provider
+
+The next step is to configure the service provider.
+
+1. Return to the WSO2 IS management console.
+
+2. Navigate to **Main** tab -> **Identity** -> **Service Providers** -> **Add**.
+
+3. Enter **pickup-dispatch** in the **Service Provider Name** text box,
+    and click **Register**.
+
+4. In the **Inbound Authentication Configuration** section, click
+    **Configure** under the **OAuth/OpenID Connect Configuration** section.
+    
+5. Enter the following value as the **Callback URL**: http://localhost.com:8080/pickup-dispatch/oauth2client
+
+    !!! Tip
+        The callback URL is the service provider URL to which the authorization codes are sent. Upon successful authentication, the browser should be redirected to this URL. 
+
+6. Click **Add**. 
+
+    !!! Tip
+        For more information on the advanced configurations
+        refer, [Configuring OAuth2-OpenID Connect Single-Sign-On](../../learn/configuring-oauth2-openid-connect-single-sign-on/)
+
+5.  Click **Register** to save the changes.  
+    Now you are sent back to the Service Providers page.
+
+----------------
+
+## Deploying the pickup-manager webapp
+
+!!! tip "Before you begin" 
+    - Install Apache Tomcat 8.x as explained in the
+    beginning of this document.
+    - Note that this sample uses OpenID Connect.
+    
+### Download the sample
+
+To be able to deploy a WSO2 Identity Server sample, you need to download
+it onto your machine first.
+
+Follow the instructions below to download a sample from GitHub.
+
+1. Navigate to [WSO2 Identity Server Samples](https://github.com/wso2/samples-is/releases).
+2. [Download](https://github.com/wso2/samples-is/releases/download/v4.1.0/pickup-manager.war) the `pickup-manager.war` file from the
+   latest release assets.
+
+### Deploy the sample web app
+
+Deploy this sample web app on a web container.
+
+1. Copy the .war file into the `           webapps          `
+    folder. For example,
+    `           <TOMCAT_HOME>/apache-tomcat-<version>/webapps          `
+    .
+2. Start the Tomcat server.
+
+To check the sample application, navigate to
+`          http://<TOMCAT_HOST>:<TOMCAT_PORT>/pickup-manager.com/home.jsp        `
+on your browser.
+
+For example,
+`                     http://localhost:8080/pickup-manager.com/home.jsp                   .         `
+
+
+### Configuring the service provider
+
+The next step is to configure the service provider.
+
+1. Return to the WSO2 IS management console.
+
+2. Navigate to **Main** tab -> **Identity** -> **Service Providers** -> **Add**.
+
+3. Enter **pickup-manager** in the **Service Provider Name** text box,
+    and click **Register**.
+
+4. In the **Inbound Authentication Configuration** section, click
+    **Configure** under the **OAuth/OpenID Connect Configuration** section.
```

Comment:
Under the **Inbound Authentication Configuration** section, click **OAuth/OpenID Connect Configuration > Configure**.